operations/node_management/Reboot_NCNs.md

```diff
@@ -138,6 +138,18 @@ The `kubectl` command is installed.
 
         If there are BGP Peering sessions that are not ESTABLISHED on either switch, refer to [Check BGP Status and Reset Sessions](../network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md).
 
+    1. Check for components that are in an failed state in CFS.
+
+        Components can be found by running the following command:
+        ```bash
+         ncn-m001# cray cfs components list --status failed
```

Comment:
The beginning of the command prompt (the letter n) should be indented to the same depth as the three backticks that surround it